cellprofiler_core/measurement/_measurements.py

```diff
@@ -106,11 +108,11 @@ def __init__(
         elif filename is None:
             fd, filename = make_temporary_file()
             is_temporary = True
-            import traceback
+            LOGGER.debug("Created temporary file %s" % filename)
 
-            logging.debug("Created temporary file %s" % filename)
-            for frame in traceback.extract_stack():
-                logging.debug("{}: ({} {}): {}".format(*frame))
+            # import traceback
```

Comment:
Can we remove this, or is it being kept for OMERO reasons? If there's a reason to keep, can we leave ourselves a note as to why?